### PR TITLE
Added CBV contrast

### PR DESCRIPTION
**Pull request by TheChymera**
As discussed https://github.com/bids-standard/bids-specification/issues/63

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -111,14 +111,13 @@ Template:
...
+        sub-<participant_label>[_ses-<session_label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>]_<contrast_label>.nii[.gz]
         sub-<participant_label>[_ses-<session_label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_mod-<label>]_defacemask.nii[.gz]
 ```
 
-Anatomical (structural) data acquired for that participant. Currently supported
-modalities include:
+Currently supported image contrasts include:
 
-| Name               | modality_label | Description                                                                                                                                                                                                                             |
+| Name               | contrast_label | Description                                                                                                                                                                                                                             |
```
Comment (chrisfilo) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Changes to the anatomical section seem tangential and I would leave them to another PR to keep the discussion focused.

**Comment by chrisfilo** [comment]
Thanks for sending this. It seems that you have a few syntax issues (see https://travis-ci.com/bids-standard/bids-specification/builds/90769181#L483). Have a look at https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md#writing-in-markdown for tips on how to deal with Markdown.

Pinging @bids-standard/bep_leads for comments.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md (hunk 1 above)
Comment (chrisfilo) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Depends how widely you propose it to apply - if for the whole spec then in definitions.

Please move these changes to a separate pull request - I expect this will raise a longer discussion which should not slow down merging changes related strictly to CBV.

**Comment by TheChymera** [comment]
1. Actually, yes. Added now.
2. No

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md (hunk 1 above)
Comment (TheChymera) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: ok

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -74,10 +74,11 @@ first "effective" echo and the center of the last "effective" echo.
...
+| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                                            |
+| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                                                                                                                                                                                                                                                   |
+| Negative Contrast           | OPTIONAL. Whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol. Values MUST be one of True, False.                  |
```
Comment (chrisfilo) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: I would make it explicit that this is a `boolean` and values need to be `true` or `false` (See https://www.w3schools.com/js/js_json_datatypes.asp). Is there a DICOM field corresponding to this field?

Comment (TheChymera) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: done, and no, no DICOM analogue.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -74,10 +74,11 @@ first "effective" echo and the center of the last "effective" echo.
...
+| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                                            |
+| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                                                                                                                                                                                                                                                   |
+| NegativeContrast           | OPTIONAL. Boolean ("true" or "false") value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol.        |
```
Comment (chrisfilo) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]:
```suggestion
| NegativeContrast           | OPTIONAL. Boolean (`true` or `false`) value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol.        |
```

Comment (TheChymera) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: I saw that this was also an alternative in the current document, but "true" and "false" here are quoted strings (like what you normally put in quotation marks in all sort of text), and not inlined code.

Comment (chrisfilo) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: I want to avoid giving the impression those values are strings not booleans in strict JSON sense.

Comment (TheChymera) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: ok

**Comment by chrisfilo** [comment]
I think you need to rebase or merge to resolve conflicts before we can merge.

**Comment by TheChymera** [comment]
@chrisfilo ping

**Comment by chrisfilo** [comment]
Thanks!
